layouts/partials/nav.html

```diff
@@ -46,22 +45,29 @@ <h2 class="title is-5">{{$printable}}</h2>
         {{- end -}}
         {{- end -}}
       </div>
-      {{ end }}
 
-      {{ if ne .Kind "taxonomy" }}
-      {{ if (isset .Site.Params "sections_right" ) }}
       <div class="nav-right">
-        {{- range $section, $printable := .Site.Params.sections_right -}}
-        {{- if $printable -}}
-        <a class="nav-item" href="{{$section}}">
-          <h2 class="title is-5">{{$printable}}</h2>
-        </a>
-        {{- end -}}
-        {{- end -}}
+        <form onsubmit="google_search(); return false;">
+            <div class="form-group form-group-lg">
+                <input
+                    type="text"
+                    class="form-control"
```

Comment:
Why are the `form-*` classes needed? They do not seem to change the style of the search box

Reply:
Probably copy-paste. I can remove that

Reply:
Done

Reply:
We also do not need the `div` element, given that we do not apply any class to it anymore